tools/cmake/mbed_greentea.cmake

```diff
@@ -1,9 +1,9 @@
 # Copyright (c) 2020-2021 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
-
 set(MBED_CONFIG_PATH ${CMAKE_CURRENT_BINARY_DIR} CACHE INTERNAL "")
 
 include(${CMAKE_CURRENT_LIST_DIR}/app.cmake)
+set(MBED_ROOT ${CMAKE_CURRENT_LIST_DIR}/../.. CACHE INTERNAL "")
```

Comment:
You've mentioned `CMAKE_CURRENT_LIST_FILE` in the commit message, but use `CMAKE_CURRENT_LIST_DIR` in the patch.

Reply:
Fixed.

Reply:
Can we use the existing `${mbed-os_SOURCE_DIR}` like we do for unit tests? Each project's source directory is defined by CMake: https://cmake.org/cmake/help/latest/variable/PROJECT-NAME_SOURCE_DIR.html

Reply:
> Can we use the existing ${mbed-os_SOURCE_DIR} like we do for unit tests? Each project's source directory is defined by CMake: https://cmake.org/cmake/help/latest/variable/PROJECT-NAME_SOURCE_DIR.html

Not possible here because the mbed-os project isn't known at this point. Each greentea test is it's own project. The mbed-os project isn't visible until after we add the subdirectory.

Reply:
Ah yea, I missed it